backend/src/main/kotlin/com/orchlib/backend/ComposerService.kt

```diff
@@ -0,0 +1,17 @@
+package com.orchlib.backend
+
+import com.orchlib.backend.database.ComposerDTO
+import com.orchlib.backend.database.DatabaseWriteResponse
+import com.orchlib.backend.database.JdbcComposerRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+@Service
+class ComposerService {
+    @Autowired
+    private lateinit var jdbcComposerRepository: JdbcComposerRepository
+
+    fun save(composer: ComposerDTO): DatabaseWriteResponse {
+        return jdbcComposerRepository.save(composer)
```

Comment:
Need to include read methods

Need to add check for already-existing composers